### DISCUSSION TITLE
subdirectory in templates

### DISCUSSION THREAD

**Discussion by gam-ragnar** [mediacms-io/mediacms#117] [Feature Requests]
To prevent over write I created a subfolder "custom" in templates where a moved "cms" and "config" folders. I do like for a check to be made to these subfolder during update. Or is there a better way to prevent overwriting changes? Btw I added "templates/custom" into local-settings as a new value in DIR

**Comment by mgogoulos** [comment]
Hi, can you describe what you're willing to achieve here? Not sure I understand!Thanks

**Comment by gam-ragnar** [comment]
Hi @mgogoulos, updates over writes any changes made to template files. I.e any changes made to about us page gets overwritten. To prevent this happening I'm moving about us out of /template/cms into template/custom/cms.

**Reply by Dan1ell** [reply]
Is it possible that when you updated, you accidentally pulled upstream MediaCMS instead of your own fork? And that reset your templates?
